scripts/otel_packaging.py

```diff
@@ -29,17 +28,19 @@ def get_instrumentation_packages():
         if not os.path.isdir(pkg_path):
             continue
 
+        error = f"Could not get version for package {pkg}"
         try:
-            version = subprocess.check_output(
-                "hatch version",
-                shell=True,
-                cwd=pkg_path,
-                universal_newlines=True,
-            )
-        except CalledProcessError as exc:
-            print(f"Could not get hatch version from path {pkg_path}")
-            print(exc.output)
-            raise exc
+            response = get(f"https://pypi.org/pypi/{pkg}/json", timeout=10)
+
+        except Exception:
+            print(error)
```

Comment:
Shouldn't we log instead of print?

Reply:
I'm fine with printing this is just a script. Logging is more susceptible to logging configuration that may produce no output.